cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `namespace` argument in the `include` function requires that the `app_name` variable is defined in the `urls.py` files of the respective apps (`cinema` and `user`). Please ensure that the `app_name` is correctly set in those files to avoid any namespace-related issues.